supabase/migrations/20260817063759_guard_app_version_r2_path.sql

```
@@ -0,0 +1,79 @@
+
+  RETURN NEW;
+END;
+$$;
+
+ALTER FUNCTION public.guard_app_version_r2_path() OWNER TO postgres;
+REVOKE ALL ON FUNCTION public.guard_app_version_r2_path() FROM PUBLIC;
+GRANT ALL ON FUNCTION public.guard_app_version_r2_path() TO service_role;
+
+DROP TRIGGER IF EXISTS guard_app_version_r2_path_trigger ON public.app_versions;
+CREATE TRIGGER guard_app_version_r2_path_trigger
+BEFORE INSERT OR UPDATE OF r2_path, owner_org, app_id, name ON public.app_versions
```

SQLFluff (4.2.2)

[error] Line is too long (82 > 80).

(LT05)